boards/arm/bbc_microbit_v2/bbc_microbit_v2.dts

```diff
@@ -64,6 +64,7 @@
 		refresh-frequency = <50>;
 		timer = <&timer4>;
 		pwm = <&pwm0>;
+		pixel-group-size = <4>;
```

Comment:
Why default to 4 and not 5, which is the full row?

Reply:
5 is not possible to use with PWM (there are only 4 channels).

Reply:
I see, there is only 4 PWM channels.